Makefile

```diff
@@ -136,8 +136,8 @@ help:
 .gopathok:
 ifeq ("$(wildcard $(GOPKGDIR))","")
 	mkdir -p "$(GOPKGBASEDIR)"
-	ln -sfnT "$(CURDIR)" "$(GOPKGDIR)"
-	ln -sfnT "$(CURDIR)/vendor/github.com/varlink" "$(FIRST_GOPATH)/src/github.com/varlink"
+	ln -sfn "$(CURDIR)" "$(GOPKGDIR)"
+	ln -sfn "$(CURDIR)/vendor/github.com/varlink" "$(FIRST_GOPATH)/src/github.com/varlink"
```

Comment:
I'll let @jhonce have the final word, but could we do this conditionally?  I.e. the new variant if on BSD, the old way if not?  Something like:  https://gist.github.com/sighingnow/deee806603ec9274fd47

Reply:
I also considered to implement an os-dependent version using ``LNFLAGS`` but imo the ``-T`` flag just makes sense if ``$(GOPKGDIR)`` is an existing directory which is already checked in the ``ifeq`` clause using the `wildcard` function.

Reply:
I am fine with this change, but, would libpod even work on BSD?  Do they have OCI runtimes?

Reply:
I am not sure about OCI runtimes for BSD but this is even necessary for building the podman-remote binaries, e.g. ``make podman-remote-darwin`` on macOS which uses BDS's variant of ``ln``.